platform/mds/mds-web/src/main/resources/webapp/js/directives.js

```diff
@@ -502,6 +502,9 @@
             link: function (scope, element, attr, ngModel) {
                 var isReadOnly = scope.$eval(attr.ngReadonly);
                 if(!isReadOnly) {
+                    angular.element(element).click( function() {
```

Comment:
For me, switching is the same as removing one class and adding a different one, but maybe my definition is wrong? I still don't get why we need to use javascript. Won't overriding the class 'ui-priority-secondary' be a better solution? Doing stuff like this in onclick just seems odd.

The current javascript solution doesn't work after I click the 'now' button.

Moreover, if this the default datepicker behavior, should we even consider this a bug?